packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -590,6 +594,16 @@ class EditableText extends StatefulWidget {
              ]
            : inputFormatters,
        showCursor = showCursor ?? !readOnly,
+       contentCommitMimeTypes = contentCommitMimeTypes.isEmpty
+            ? onContentCommitted == null ? <String>[] : <String>[
+          'image/png',
+          'image/bmp',
+          'image/jpg',
+          'image/tiff',
+          'image/gif',
+          'image/jpeg',
+          'image/webp'
+       ] : contentCommitMimeTypes,
```

Comment:
Nit: I think we should also set this default in the same way in CupertinoTextField and TextField, and document that's what the default is there too.

Reply:
Also, maybe we could reuse this logic if it works out.  I noticed that it's duplicated below in the default values method.  Maybe a static method on EditableText?